analyses/focal-cn-file-preparation/01-prepare-cn-file.R

```diff
@@ -263,13 +314,40 @@ cnv_no_xy <- cnv_df %>%
 
 # Merge and annotated no X&Y
 autosome_annotated_cn <- process_annotate_overlaps(cnv_df = cnv_no_xy,
-                                                   txdb_exons = tx_exons) %>%
+                                                   cds_object = cds_gr) %>%
```

Comment:
Ah. It's a `GenomicRanges` object it needs. Can you change the function documentation to say that?